crates/job-queue-libsql/Cargo.toml

```diff
@@ -12,10 +12,13 @@ workspace = true
 anyhow = "1"
 async-trait = "0.1"
 job-queue = { path = "../job-queue" }
-libsql = { git = "https://github.com/tursodatabase/libsql", rev = "9b04f1d57d8d098852fed5cd15d6526d89392c4b", default-features = false, features = ["core", "serde", "replication"] }
+libsql = { version = "0.9", default-features = false, features = ["core", "serde", "replication"] }
 serde = "1"
 time = { version = "0.3", default-features = false, features = ["std"] }
 tracing = "0.1"
 typed-builder = "0.19"
 ulid = { version = "1.1", features = ["uuid"] }
 uuid = "1.11"
+
+[patch.crates-io]
+libsql = { git = "https://github.com/allan2/libsql", branch = "ffi-windows-cp" }
```

Comment:
issue: dependency patches not found in the workspace Cargo.toml manifest are ignored. This should be moved to the toplevel Cargo.toml

suggestion: Also include the revision (commit hash).